### PR TITLE
packs doc

### PR DESCRIPTION
**Pull request by anara123**
## Status
Ready

## Related Issues
fixes: link to the issue

## Description
General guidelines to write pack README/documentation 

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/integrations/pack-docs.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+id: pack-docs
+title: Pack Documentation
+---
+
+:::note
+These guidelines are mainly for packs containing some use case, not a single integration.
```
Comment (anara123) [inline docs/integrations/pack-docs.md:7]: @kirbles19 I want to explain here that these guideline are mainly for packs with use cases/meat on them. Packs with single integration don't need/require to write pack documentation

Comment (glicht) [reply docs/integrations/pack-docs.md:7]: Note sure we need this.

**Comment by glicht** [comment]
@anara123 we need an overview explaining the different doc options (description vs README). I'll try to add something.

**Comment by glicht** [comment]
@kirbles19 preview available at: https://deploy-preview-201--demisto-content-docs.netlify.app/docs/integrations/pack-docs

File: docs/integrations/pack-docs.md (hunk 1 above)
Comment (glicht) [reply docs/integrations/pack-docs.md]: I removed this in my commit.

**Comment by anara123** [comment]
@glicht looks good to me

**Pull request merged**